consumers/email-consumer/tests/test_email_consumer.py

```
@@ -0,0 +1,97 @@
+        "email": "test@example.com",
+        "verification_code": "123456",
+        "username": "testuser",
+        "user_id": "user123",
+    }
+
+    email_consumer.process_message(message)
+    mock_send_email.assert_called_once_with(**message)
+
+
+@patch.object(EmailConsumer, "send_verification_email")
+def test_process_message_failure(mock_send_email, email_consumer):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_process_message_failure`

Add return type annotation: `None`

(ANN201)